packages/core/data-transfer/src/strapi/providers/local-source/assets.ts

```diff
@@ -1,32 +1,100 @@
 import { join } from 'path';
-import { readdir, stat, createReadStream } from 'fs-extra';
-import { Duplex } from 'stream';
+import https from 'https';
+import { stat, createReadStream, ReadStream } from 'fs-extra';
+import { Duplex, PassThrough } from 'stream';
 
-import type { IAsset } from '../../../../types';
+import type { IAsset, IFile } from '../../../../types';
 
-const IGNORED_FILES = ['.gitkeep'];
+function getFileStream(filepath: string, isLocal = false): PassThrough | ReadStream {
+  if (isLocal) {
+    return createReadStream(filepath);
+  }
 
+  const readableStream = new PassThrough();
+
+  https
```

Comment:
Why do we need this library?

Reply:
to get the a file form a en external provider

Reply:
Sure, but why don't we reuse a library that we already have (like axios for instance) instead of going through a new one? Does this library do smth than the others don't?

Reply:
Damn, I don't know why I was persuaded I saw this added to the package.json and thought "wtf is this new library?". My bad :bow: